### DISCUSSION TITLE
Rights and Restrictions + Leases   - investigate what users want /need to see to help workflows

### DISCUSSION THREAD

**Discussion by honorcb** [guardian/grid#3574] [Ideas]
Currently when an image has a lease set,  does the view make this obvious to the user, when they do a search for an image to go with a story?
Case 1
Have a image that requires a fee to be paid,  so has the Right and Restriction "Chargeable /supplied on spec"
This is the only image for a story
The fee is paid to use the image for a "date" for this story.  
Buyer of  the rights, set the  image lease to Can Crop on "date"

Currently this does not allow the image to be visible in the "Free- to use only" selection,
nor does it change the colour of the flag. 
If the Buyer adds "restriction" to this image, in the Rights and Restriction ,  this change the £ to amber,
 But Only if you choose to untick the "Free to use only" 
the expiry of the lease "can crop " date,   does not change the colour back to red.

Case 2

A picture X has been found on Social Media,  for a new story being written
Add  image X  with Rights and Restrictions  "Social Media"   
Currently GRID/Images set X with an Amber flag and a restriction note about checking usage with Senior Editor

In the BBC System Elvis,  this image X would be added with similar restriction,  but is flagged RED
When the Social Media Image  X   has usage agreed Senior Editor ,  it is flagged AMBER
 

Basic rule  Keep it Simple - 
 
The variations on  usage rights when linked to leases is large,   I have a long  list of Rights and Restriction,  with the various options,  for the rights and restriction and the lease options.  Can these variations be mapped to a few icons so users know , when looking at a search result  before selecting the image

- they cannot crop, ( use ) before the open the image
-Social Media image has been approved by a Senior Editor
-An image that cannot be syndicated,  and has a usage restriction message

The usage is currently  indicated by four icons ?

!  Red exclamation mark   NO RIGHTS 
£  Requires payment  
Flag - item has restriction set in the Rights and Restrictions.
¬ Crop icon,  when an image has been cropped

Would these four ,  if we can strike them thorough when the condition is change  giving us eight to view, cover most of the options?  The one I am less certain about is how you indicate the image cannot be syndicated.

**Comment by paperboyo** [comment]
> nor does it change the colour of the flag.

This is fixed by https://github.com/guardian/grid/pull/3854. This does not cover Deny leases.

Visibility of certain red/restricted images within Free to use searches is indeed static (does not depend on Lease status).

**Comment by honorcb** [comment]
was reading https://github.com/guardian/grid/issues/4282  and https://github.com/guardian/grid/issues/3860

I wondered is there any search you can do to find a type of lease?  I know that I can search +has:leases  but if I want to know if images in a search have  an  allow cropping  or deny cropping leases,  or  has allow syndication or  deny syndication  set against images ,  to exclude them is there a way to easily to do this using a chip?  Just we have groups of users who create content to be syndicated,  so it would be great if we could help then by filter out those they will not be able to use as they have a deny syncication lease set.

**Reply by honorcb** [reply]
"leases": {
      "data": {
        "leases": [.....

          "access": "deny-syndication",
           "active": true
is what I see for images that have deny syndication active ,   so wondered if you could start with the has:leases and add the detail you want to find  or can you search on  syndicationStatus: "unsuitable"

**Reply by paperboyo** [reply]
I agree searching here should be better. My understanding is that even when using some things in the API in search, it won’t work well. +has:leases, but is pretty basic: it shows all leases: allow, deny, syndication and expired too. We would need to do things under the hood to be able to search for ***effective** (active) Allow cropping leases (where there is no active Deny lease).  So:
https://media.gutools.co.uk/search?query=has:leases&nonFree=true (to show all)
and
https://media.test.dev-gutools.co.uk/search?query=leases.leases.access:allow-use&nonFree=true (chip displays weirdly, will show images with Allow cropping leases. But will also show images where those leases are expired and images where those leases are overwritten by Deny cropping…)

For syndication specifically, searching for syndication status can be a workaround. These rae not in the UI/chips at all, but
https://media.gutools.co.uk/search?nonFree=true&syndicationStatus=blocked
will do (displays red dollar signs for us: images with Deny syndication lease)
There are other `syndicationStatus` searches too. I believe you are adding one more status.

But yeah: searching for specific leases and their effective state could be improved. Two linked issues not directly related, but important too.
